app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,15 @@
 <body>
   <nav>
   <div class="nav-wrapper">
+      <% if session[:user_id] %>
```

Comment:
In categories/index.html.erb, we have a link for creating account if there is no current user but here we display a logout based on there being a session[:user_id]. I think we should try to be consistent in both areas. What do you think?